### PR TITLE
Populate form values

### PR DESCRIPTION
**Pull request by lhalaa**
This change includes the following:
- Move submission to a separate controller
- Update composer
- Move template to namespaced location
- Remove deprecated `Member::currentUser()`
- Disable canCreate for field and form submissions
- Update tests

### CODE REVIEW

File: src/controllers/PartialUserFormController.php [hunk 1]
```diff
@@ -140,28 +50,44 @@ public function partial(HTTPRequest $request)
...
+                    $content = preg_replace(
+                        '/(<p[^>]*>)?\\$UserDefinedForm(<\\/p>)?/i',
+                        $formEscapedForRegex,
+                        $this->Content
+                    );
+                    return $this->customise([
+                        'Content' => DBField::create_field('HTMLText', $content),
+                        'Form' => '',
+                        'PartialLink' => $partial->getPartialLink()
+                    ])->renderWith(['Firesphere\PartialUserforms\PartialUserForm', 'Page']);
```
Comment (Firesphere) [inline src/controllers/PartialUserFormController.php]: You should be able to use `PartialUserForm::class, Page::class` here

File: src/controllers/PartialUserFormController.php [hunk 2]
```diff
@@ -140,28 +50,44 @@ public function partial(HTTPRequest $request)
...
-                'Title' => $record->Title,
-                'Breadcrumbs' => $record->Breadcrumbs(),
-                'Content' => $this->obj('Content'),
-                'Form' => $controller->Form(),
-                'Link' => $partial->getPartialLink()
-            ])->renderWith(['PartialUserForm', 'Page']);
+                'Content' => DBField::create_field('HTMLText', $this->Content),
+                'Form' => $form,
+                'PartialLink' => $partial->getPartialLink()
+            ])->renderWith(['Firesphere\PartialUserforms\PartialUserForm', 'Page']);
```
Comment (Firesphere) [inline src/controllers/PartialUserFormController.php]: Same as above :)

File: src/models/PartialFormSubmission.php [hunk 3]
```diff
@@ -216,4 +212,21 @@ public function generateKey($token)
...
+    /**
+     * Get the partial fields in array
+     *
+     * @return array
+     */
+    public function getFieldList()
+    {
+        $list = [];
+        if ($this->PartialFields()->exists()) {
+            foreach ($this->PartialFields() as $field) {
```
Comment (Firesphere) [inline src/models/PartialFormSubmission.php]: Could you use `$this->PartialFields()->map('Name', 'Value')->toArray()` here?

**Review by Firesphere — CHANGES_REQUESTED** [review]

File: src/controllers/PartialUserFormController.php (hunk 1 above)
Comment (lhalaa) [reply src/controllers/PartialUserFormController.php]: We don't have `PartialUserForm::class` so I used the controller class instead

File: src/controllers/PartialUserFormController.php [hunk 4]
```diff
@@ -140,28 +51,46 @@ public function partial(HTTPRequest $request)
...
-                'Title' => $record->Title,
-                'Breadcrumbs' => $record->Breadcrumbs(),
-                'Content' => $this->obj('Content'),
-                'Form' => $controller->Form(),
-                'Link' => $partial->getPartialLink()
-            ])->renderWith(['PartialUserForm', 'Page']);
+                'Content' => DBField::create_field('HTMLText', $this->Content),
+                'Form' => $form,
+                'PartialLink' => $partial->getPartialLink()
+            ])->renderWith([static::class, Page::class]);
```
Comment (Firesphere) [inline src/controllers/PartialUserFormController.php:93]: 👍

**Review by Firesphere — APPROVED** [review]

**Pull request merged**